### ISSUE TITLE
[Tensorrt]: Cannot set value field in weight class

### ISSUE THREAD

**Issue by lpaatero** [bytedeco/javacpp-presets#698]
I am attempting to construct new network using tensorrt API.

When doing this, I have not been able to find any way to set values field in Weights class.
In particular, there is no values(Pointer) method in generated Weights java class.

Weights class in .h is defined as:
> class Weights
> {
> public:
>      DataType type;      
>      const void* values; 
>      int64_t count;      
> };

This seems to prevent me from creating any network at all.

**Comment by saudet** [comment]
I see what the issue is. Adding the following to `presets/nvinfer.java` fixes this:
```java
.put(new Info("nvinfer1::Weights::values").javaText("public native @Const Pointer values(); public native Weights values(Pointer values);"))
```

**Comment by saudet** [comment]
I've included the fix in commit https://github.com/bytedeco/javacpp-presets/commit/18a01baabf88811bcae67ff3e136c747ab0bba71. Please give it a try with 1.4.5-SNAPSHOT in a few hours when the builds are ready: http://bytedeco.org/builds/

**Comment by lpaatero** [comment]
Thanks for fast fix!
This solved the problem, I have got a toy example working.

**Comment by saudet** [comment]
Fix included in version 1.5. Thanks for reporting!
